src/osgPlugins/OpenFlight/expPrimaryRecords.cpp

```diff
@@ -688,13 +688,13 @@ FltExportVisitor::writeLightPoint( const osgSim::LightPointNode* lpn )
     float32 horizLobe( 360.f );
     float32 vertLobe( 360.f );
     float32 lobeRoll( 0.f );
-    const osgSim::DirectionalSector* dirSector = dynamic_cast< osgSim::DirectionalSector* >( lp0._sector.get() );
```

Comment:
Looks like the 3.6 version fixed the same problem we did but in a better way. Let's use the 3.6 version.

Reply:
Reverted.